src/index.html

```diff
@@ -7,8 +7,39 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css" rel="stylesheet">
 </head>
 <body>
-  <app-root></app-root>
+  <div>
```

Comment:
Move header into separate component that used in root component

Reply:
Changed